stat/distuv/f_test.go

```diff
@@ -90,4 +91,62 @@ func testF(t *testing.T, f F, i int) {
 	checkExKurtosis(t, i, x, f, 1e-1)
 	checkSkewness(t, i, x, f, 5e-2)
 	checkQuantileCDFSurvival(t, i, x, f, 5e-3)
+	checkMode(t, i, x, f, 2e-2, 3e-2)
```

Comment:
What are the magic numbers based on?

Reply:
Testing `Mode` empirically requires discretising the values (binning). The first argument is the size of the bin. F distribution's support extends over a range of the order of 1-10, so 0.02 seems like a decent size. The second argument is the tolerance. It's larger than the bin size due to finite sample size.

Reply:
Thanks